### PR TITLE
Adding prelude and update changelog commands for installer

### PR DESCRIPTION
**Pull request by ghost**
### What does this PR do?

Add changelog for installer changes

### Motivation

Better understand and document the installer changes

### Additional Notes

This change doesn't have the release note generated or the code tagging. It will be done when we are ready to generate the real release note.

### Describe your test plan

You should have no uncommitted changes on the private branch.

Create dummy release notes, commit, then tag the commit. The tag must be in install-x.y.z format.

```
reno --rel-notes-dir ./releasenotes-install new test1
git add ./releasenotes-install/notes/*.yaml
git commit -m "test"
git tag -a -f install-120.0.0 -m "install-120.0.0"
```

Generate release notes:

`inv release.update-install-changelog 120.0.0`

Should be no failure and it will update the release note and commit it (without push).

### CODE REVIEW

**Review by albertvaka — COMMENTED** [review]
I'm being nitpicky here, but I would rename "install" to "installscript" since we have more than once install method (eg: Windows MSI installer) and these notes are only for one of those methods, the install script.

**Comment by mikezhu-dd** [comment]
> I'm being nitpicky here, but I would rename "install" to "installscript" since we have more than once install method (eg: Windows MSI installer) and these notes are only for one of those methods, the install script.

Will do the change, new PR coming in 1 hr. Thanks.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -69,6 +69,7 @@
 # Assigning them to a group that doesn't exist means nobody will receive notifications for them, but
 # that should be fine since rarely we make PRs that only change those files alone.
 /releasenotes/                          @DataDog/do-not-notify
+/releasenotes-install/                  @DataDog/do-not-notify
```
Comment (albertvaka) [inline .github/CODEOWNERS:72]:
```suggestion
/releasenotes-installscript/                  @DataDog/do-not-notify
```

File: pyproject.toml [hunk 2]
```diff
@@ -14,6 +14,7 @@ exclude = '''
     | \.git
     | docs
     | releasenotes
+    | releasenotes-install
```
Comment (albertvaka) [inline pyproject.toml:17]:
```suggestion
    | releasenotes-installscript
```

**Review by albertvaka — APPROVED** [review]

**Comment by albertvaka** [comment]
To be merged after the code freeze.

**Pull request merged**